Tasks/Common/compression-common/make.json

```diff
@@ -0,0 +1,10 @@
+{
+    "externals": {
+        "archivePackages": [
+            {
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/7zip/1/7zip.zip",
```

Comment:
Why are we packaging 7zip with each task version and add 1MB.  The tool lib has 7z support.

Reply:
We got it down to 45K in the lib

Reply:
Agreed, we should use tool lib here.  It looks like we must have appropriated the code from tool lib so we could specify the extraction directory.  I will send out a PR to tool lib to support this and then use tool lib.

Reply:
Agreed, we should use tool lib here. It looks like we must have appropriated the code from tool lib so we could specify the extraction directory. I will send out a PR to tool lib to support this and then use tool lib.